### PR TITLE
Reagent Containers : The 3/4thening & Contained Spritening (and other related food things)

### PR DESCRIPTION
**Pull request by alsoandanswer**
Notable changes : Contained Spritizes *all* reagent containers, and reagent fillings.

changes:
  - imageadd: "Ports 3/4 bottle, soda can, kitchen condiments, ingredients, reagent containers from /tg/. Updates unique Aurora sprites to this standard as well."
  - imageadd: "Updates beaker, droppers, pipettes and chemical mixers and chemmasters."
  - imageadd: "New chemical cartridge, flask, vacuum flask and shaker sprites."
  - rscadd: "Adds champagne popping and sabrage."
  - rscadd: "Ports rice beer, beer cans and melon soda."  **<-- (for konyang!)**
  - tweak: "Rice beer recipe is now 10 rice, 5 universal enzyme (catalyst), Sake is now 5 rice beer, 5 moss and 5 universal enzyme (catalyst)." **<-- (for konyang!)**
  - tweak: "Adds light beer, recipe 5 beer and 5 water. Tweaks recipe of beer to be 5 flour and 5 universal enzyme."
  - rscadd: "Hooch now has it's own bottle."
  - backend: "More soda can yoke variants."
  - backend: "Contained spritizes reagent containers,  surgical tools and stethoscope."
  - imageadd: "Improves sprites for syringes, autoinjectors, DNA injectors, Lazarus injectors. They should look better and be more consistent (all with a 45 degree tilt)"
  - tweak: "Replaced 'milosoup' with miso soup, updates the sprite." (by atteria!)
  - imageadd: "New teapot sprite." (by atteria!)

### CODE REVIEW

File: code/modules/reagents/reagent_containers/food/drinks/bottle.dm [hunk 1]
```diff
@@ -289,6 +330,124 @@
...
+	balloon_alert(user, "preparing to swing...")
+	if(!do_after(user, 2 SECONDS, src)) //takes longer because you are supposed to take the foil off the bottle first
+		return
+
+	///The bonus to success chance that the user gets for being a command role
+	var/command_bonus = (user.mind?.assigned_role in command_positions) ? 20 : 0
+
+	var/job_bonus = user.mind?.assigned_role == "Bartender" ? 25 : 0
+
+	//calculate success chance. example: captain's sabre - 15 force = 75% chance
```
Comment (NonQueueingMatt) [inline code/modules/reagents/reagent_containers/food/drinks/bottle.dm:369]: outdated comment

**Review by NonQueueingMatt — APPROVED** [review]

File: code/modules/reagents/reagent_containers/food/drinks/bottle.dm [hunk 2]
```diff
@@ -366,6 +524,14 @@
 	center_of_mass = list("x"=16, "y"=4)
 	reagents_to_add = list(/singleton/reagent/alcohol/wine = 100)
 
+/obj/item/reagent_containers/food/drinks/bottle/pwine
+	name = "Chip Getmore's Velvet"
```
Comment (SleepyGemmy) [inline code/modules/reagents/reagent_containers/food/drinks/bottle.dm:528]:
```suggestion
	name = "\improper Chip Getmore's Velvet"
```

File: code/modules/reagents/reagent_containers/food/drinks/bottle.dm [hunk 3]
```diff
@@ -743,6 +884,15 @@
...
 	reagents_to_add = list(/singleton/reagent/alcohol/wine/valokki = 100)
 
+/obj/item/reagent_containers/food/drinks/bottle/gibsonhooch
+	name = "Fallanland Hooch"
+	desc = "For the few times that's socially acceptable to be bare-chested in public - at a sporting event, blasted on cheap grain alcohol."
+	desc_extended = DRINK_FLUFF_GETMORE
+	icon_state = "hoochbottle"
+	center_of_mass = list("x"=16, "y"=8)
+
+	reagents_to_add = list(/singleton/reagent/alcohol/gibsonhooch = 100)
```
Comment (SleepyGemmy) [inline code/modules/reagents/reagent_containers/food/drinks/bottle.dm:894]:
```suggestion
/obj/item/reagent_containers/food/drinks/bottle/gibsonhooch
	name = "Fallanland Hooch"
	desc = "For the few times that's socially acceptable to be bare-chested in public - at a sporting event, blasted on cheap grain alcohol."
	desc_extended = DRINK_FLUFF_GETMORE
	icon_state = "hoochbottle"
	center_of_mass = list("x"=16, "y"=8)
	reagents_to_add = list(/singleton/reagent/alcohol/gibsonhooch = 100)
```

**Comment by alsoandanswer** [comment]
!review

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**